src/api/Jalasoft.TeamUp.Projects.API/Controllers/PostProject.cs

```diff
@@ -29,10 +30,29 @@ public PostProject(IProjectsService postProjectService)
         public async Task<IActionResult> CreateProject(
             [HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = "v1/projects")] HttpRequest req)
         {
-            string requestBody = await new StreamReader(req.Body).ReadToEndAsync();
-            var data = JsonConvert.DeserializeObject<Project>(requestBody);
-            var result = this.postProjectService.PostProject(data);
-            return new CreatedResult("v1/projects/:id", result);
+            try
+            {
+                string requestBody = await new StreamReader(req.Body).ReadToEndAsync();
+                var data = JsonConvert.DeserializeObject<Project>(requestBody);
+                var result = this.postProjectService.PostProject(data);
+                return new CreatedResult("v1/projects/:id", result);
+            }
+            catch (ProjectsException e)
+            {
+                return new ContentResult
+                {
+                    StatusCode = e.StatusCode,
+                    Content = e.ProjectsErrorMessage,
+                };
+            }
+            catch (System.Exception)
+            {
+                return new ContentResult
+                {
+                    StatusCode = 500,
```

Comment:
You need to return an object with this format:
{
"message" : "message of error"
}